Tutorial/Dialogue/od_010409037.csv

```diff
@@ -0,0 +1,10 @@
+text01#0,"""ご苦労だった。<br>期待を集めるだけあって<br>基礎はしっかりしているな。"""
+text02#0,"""だが私に言わせれば、その程度なら<br>掃いて捨てるほどいる。本当に<br>伸びるかどうかはまったく別の問題だ。"""
+text03#0,"""とはいえ君はまだまだ<br>伸びそうな気がする。"""
+text04#0,"""ところで、スキルのジャストガードは<br>活用しているかな？"""
+11_voice_NPC_5054_100#0,"""ハンターとして生きていくなら<br>習得して損はない。<br>では、また会おう！"""
+text100#0,"""About \""Just Guard\"""
```

Comment:
` od_010409037.csv``text100` is missing `""`  at the end

Reply:
fixed, screaming

Reply:
line text100 is still odd